### PR TITLE
[RW-8228][risk=no] Bump runtime min disk size

### PR DESCRIPTION
**Pull request by calbach**
According to Leo team, this may be causing some creations to time out.

Refactor min disk size to a central location.

### CODE REVIEW

File: ui/src/app/utils/runtime-presets.ts [hunk 1]
```diff
@@ -18,7 +22,7 @@ export const runtimePresets: {
       // TODO: Support specifying toolDockerImage here.
       gceConfig: {
         machineType: 'n1-standard-4',
-        diskSize: 100,
+        diskSize: MIN_DISK_SIZE_GB,
```
Comment (jmthibault79) [inline ui/src/app/utils/runtime-presets.ts:25]: this is a nice change, thanks.  What about a const for `n1-standard-4` as well?

File: ui/src/testing/stubs/runtime-api-stub.ts [hunk 2]
```diff
@@ -8,18 +8,18 @@ import {
...
-import { diskApi } from 'app/services/swagger-fetch-clients';
+} from 'app/utils/machines';
 
 import { stubNotImplementedError } from 'testing/stubs/stub-utils';
 
 import { DiskApiStub, stubDisk } from './disk-api-stub';
 
 export const defaultGceConfig = (): GceConfig => ({
-  diskSize: 120,
+  diskSize: MIN_DISK_SIZE_GB + 30,
```
Comment (jmthibault79) [inline ui/src/testing/stubs/runtime-api-stub.ts:22]: Hmm...  I see it's equal to `DATAPROC_WORKER_MIN_DISK_SIZE_GB` but this is not Dataproc so it makes sense for us not to use that const.  But where does the +30 come from?

**Comment by jmthibault79** [comment]
Additional thoughts about the runtime config window:
* it correctly prevented me from changing my disk size to 119 GB
* do we need to update the costs ?

File: ui/src/app/utils/runtime-presets.ts (hunk 1 above)
Comment (calbach) [reply ui/src/app/utils/runtime-presets.ts:26]: Done - actually we already had this constant.

File: ui/src/testing/stubs/runtime-api-stub.ts (hunk 2 above)
Comment (calbach) [reply ui/src/testing/stubs/runtime-api-stub.ts:24]: Added a comment here

File: ui/src/app/pages/analysis/runtime-panel.spec.tsx [hunk 3]
```diff
@@ -353,9 +356,9 @@ describe('RuntimePanel', () => {
     // TODO(RW-7152): This test is incorrectly depending on "default" values in runtime-panel, and
     // not general analysis. Ensure this test passes for the right reasons when fixing.
     const computeDefaults = wrapper.find('#compute-resources').first();
-    // defaults to generalAnalysis preset, which is a n1-standard-4 machine with a 100GB disk
+    // defaults to generalAnalysis preset, which is a n1-standard-4 machine with a 120GB disk
```
[the remark this answers is not in the document]
Comment (calbach) [reply ui/src/app/pages/analysis/runtime-panel.spec.tsx]: Good point, updated a bunch of these test comments

**Review by calbach — COMMENTED** [review]
> [Unrelated to this PR] It did not stop me from creating a runtime even though I'm out of Initial Credits. Do we have a ticket for this?

I suspect it may be related to https://precisionmedicineinitiative.atlassian.net/browse/RW-6280

> do we need to update the costs ?

Which costs do you mean? All runtime cost indicators in the UI should be dynamically computed based on the provided config.

**Comment by jmthibault79** [comment]
> I suspect it may be related to https://precisionmedicineinitiative.atlassian.net/browse/RW-6280
I think you're right - that workspace has ACTIVE billing status, and it should not.

> All runtime cost indicators in the UI should be dynamically computed based on the provided config.
Excellent, that's what I was hoping

**Pull request merged**